public/locales/en.json

```diff
@@ -171,21 +189,36 @@
     "subsamplingSliderTootlip": "We subsample transcripts as the viewer zooms out for performance. Density is preserved when subsampling"
   },
   "viewer": {
+    "clearSelectionDescription": " Are you sure you want to remove all {{polygonCount}} polygons? This action cannot be undone.",
```

Comment:
Dynamic value does not work

<img width="809" height="409" alt="image" src="https://github.com/user-attachments/assets/070f090a-4fab-464a-991a-cd55138c15d9" />